packages/next/build/swc/src/lib.rs

```diff
@@ -0,0 +1,65 @@
+#![recursion_limit = "2048"]
+//#![deny(clippy::all)]
+
+#[macro_use]
+extern crate napi_derive;
+/// Explicit extern crate to use allocator.
+extern crate swc_node_base;
+
+use backtrace::Backtrace;
+use napi::{CallContext, Env, JsObject, JsUndefined};
+use std::{env, panic::set_hook, sync::Arc};
+use swc::{Compiler, TransformOutput};
+use swc_common::{
+    self,
+    errors::{ColorConfig, Handler},
+    sync::Lazy,
+    FilePathMapping, SourceMap,
+};
+
+mod hook_optimizer;
+mod transform;
+mod util;
+
+static COMPILER: Lazy<Arc<Compiler>> = Lazy::new(|| {
+    let cm = Arc::new(SourceMap::new(FilePathMapping::empty()));
+    let handler = Arc::new(Handler::with_tty_emitter(
```

Comment:
Note: This will emit errors to stderr.
It's a mistake I made while building swc and it will be changed to exception in swc v2.